kroxylicious/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -155,10 +166,23 @@ else if (KQueue.isAvailable()) {
             bindFuture = serverBootstrap.bind(proxyPort);
         }
         acceptorChannel = bindFuture.sync().channel();
-        //
         return this;
     }
 
+    private void optionallyStartMetricsListener(EventLoopGroup bossGroup,
+                                                EventLoopGroup workerGroup,
+                                                Class<? extends ServerChannel> channelClass)
+            throws InterruptedException {
+        if (metricsConfig != null && metricsConfig.getPrometheus() != null && metricsConfig.getPrometheus().isEnabled()) {
```

Comment:
Personally I prefer putting the operands on their own line, that way it's much easier not to skip over one (it can also make debugging easier too).

Reply:
In the refactor this has changed a bit, I've split it into operand per line but also introduced null objects and an Optional so we can do `adminHttpConfig.getEndpoints().maybePrometheus().isPresent()`